ui/src/layout/Navigation.tsx

```diff
@@ -54,7 +54,7 @@ class Navigation extends Component<
     public render() {
```

Comment:
I'm not sure if I like the sorting by name. I'd rather have the applications order fully customizable via drag&drop.